.nvmrc

```diff
@@ -1 +1 @@
-lts/*
+14
```

Comment:
Node LTS used to be `14`, but now it's `16`. It seems that Gutenberg [still uses](https://github.com/WordPress/gutenberg/blob/408bcd819fb60cfffe7fa10f57c86be0d5693ec6/.nvmrc) `14`

Reply:
interesting! I haven't heard of this `lts` before.